samples/tutorial/BindingToCommandHandlers.md

```diff
@@ -5,35 +5,40 @@
 
 The simplest way to bind command line input is to set the `Command.Handler` property. The `System.CommandLine` model binder will look at the options and arguments for the command and attempt to match them to the parameters of the specified handler method. The default convention is that parameters are matched by name, so in the following example, option `--an-int` matches the parameter named `anInt`. Matching ignores hyphens (and other option prefixes, such as `'/'`) and is case insensitive.
 
-```cs --source-file ./src/Binding/HandlerBindingSample.cs --project ./src/Binding/Binding.csproj --region MultipleArgs 
+```cs
```

Comment:
📝 `cs` doesn't seem like the correct language selection for this block. With that said, I'm not sure what the _right_ language to use is either.

Reply:
@sharwell , I agree.
In that case I would suggest:
1- Use `console`, which is already being used in _README.md_.
2- Leave it empty, which give the same _look & feel_.